### ISSUE TITLE
[React 19] [SUGGESTION]  Consistency: Rename useFormStatus().pending to useFormStatus().isPending

### ISSUE THREAD

**Issue by hnordt** [facebook/react#31747]
This isn’t a bug report or a question, apologies if this isn’t the right place to post.

Throughout the documentation, conventions like adding prefixes to booleans (e.g., isPending for startTransition, useActionState, etc.) are consistently followed.

To keep things consistent, could useFormStatus().pending be renamed to useFormStatus().isPending?

To prevent a breaking change, you could deprecate pending and introduce isPending as an alternative.

**Comment by kevinuehara** [comment]
I agree with @hnordt point, I think it leaves the property in the standards of other technologies like React Query.

**Comment by Arunkumar774** [comment]
Hi, I’d like to work on this suggestion as my first open-source contribution. Could you please assign it to me? I’ll make sure to follow the contribution guidelines and keep the changes aligned with the project's standards. Thank you!

**Comment by hnordt** [comment]
Bump
